Blackmarket Retouch

Kesiath opened the pull request:
## About the PR
A few changes to a couple blackmarket ships; Removing expedition capability from the Infiltrator and adding the pirate cannon guns to the Schooner.

## Why / Balance
Getting ships more in-line with blackmarket standards as we prepare for changes regarding piracy in the future.

**Changelog**
:cl:
- remove: Removed expedition capability from the Infiltrator
- tweak: Added the pirate cannons to the Schooner

dvir001 commented:
> ## About the PR
> 
> A few changes to a couple blackmarket ships; Removing expedition capability from the Infiltrator and adding the pirate cannon guns to the Schooner.
> ## Why / Balance
> 
> Getting ships more in-line with blackmarket standards as we prepare for changes regarding piracy in the future.
> 
> **Changelog** 🆑
> 
>     * remove: Removed expedition capability from the Infiltrator
> 
>     * tweak: Added the pirate cannons to the Schooner

Bruh i reworked them all on the upstream update

Kesiath commented:
> > ## About the PR
> > A few changes to a couple blackmarket ships; Removing expedition capability from the Infiltrator and adding the pirate cannon guns to the Schooner.
> > ## Why / Balance
> > Getting ships more in-line with blackmarket standards as we prepare for changes regarding piracy in the future.
> > **Changelog** 🆑
> > ```
> > * remove: Removed expedition capability from the Infiltrator
> > 
> > * tweak: Added the pirate cannons to the Schooner
> > ```
> 
> Bruh i reworked them all on the upstream update

oopsie doodle
